site/app/views/grading/ElectronicGraderView.php

```diff
@@ -1556,7 +1558,7 @@ public function renderPeerPanel(GradedGradeable $graded_gradeable, int $display_
         return $this->core->getOutput()->renderTwigTemplate("grading/electronic/PeerPanel.twig", [
                 "gradeable_id" => $gradeable->getId(),
                 "is_ta_grading" => $gradeable->isTaGrading(),
-                "anon_id" => $graded_gradeable->getSubmitter()->getAnonId(),
+                "anon_id" => $graded_gradeable->getSubmitter()->isTeam() ? $graded_gradeable->getSubmitter()->getAnonId() : $graded_gradeable->getSubmitter()->getAnonId($graded_gradeable->getGradeableId()),
```

Comment:
This ternary operator of if its a team do 2 separate things seems to be used quite often. Could we perhaps add a function in the Submitter class to get the respective team or anon id based on which it is to reduce these a bit?

Reply:
A check inside `Submitter::getAnonId()` was already added, so all these ternary operators have been removed and replaced with a call similar to `$graded_gradeable->getSubmitter()->getAnonId($graded_gradeable->getGradeableId())`.